### ISSUE TITLE
Remove alicdn.com link and replace web font with SVG

### ISSUE THREAD

**Issue by jhildenbiddle** [imyelo/docsify-pagination#2]
This plugin injects styles two different ways:

1. By generating a `<style>` element and adding the contents of [`stylesheet.css`](https://github.com/imyelo/docsify-pagination/blob/master/src/stylesheet.css).
1. By generating a `<link>` element that link to a [stylesheet on alicdn.com](http://at.alicdn.com/t/font_609086_tain7fbkcmzpvi.css).

My concern is with the generated `<link>`. Inserting styles using a `<link>` creates creates an additional request, and in this particular case the linked stylesheet contains a `@font-face` declaration which (depending on the browser) will create a second request for the font file. The end result is two additional requests created for less than 2k of data.

Here's an alternative approach:

1. Remove the `<link>` to alicdn.com
2. Replace web font iconography with SVGs

   ```css
   .pagination-item--previous .pagination-item-label {
     display: inline-block;
     padding-left: 20px;
     background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='12.8' viewBox='0 0 8 12.8'%3E%3Cpath d='M6.75 1.25l-5.5 5.15 5.5 5.15' stroke-width='1.25' stroke='rgb%280, 0, 0%29' fill='none' vector-effect='non-scaling-stroke'/%3E%3C/svg%3E") no-repeat left center;
   }
   
   .pagination-item--next .pagination-item-label {
     display: inline-block;
     padding-right: 20px;
     background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='12.8' viewBox='0 0 8 12.8'%3E%3Cpath d='M1.25 1.25l5.5 5.15 -5.5 5.15' stroke-width='1.25' stroke='rgb%280, 0, 0%29' fill='none' vector-effect='non-scaling-stroke'/%3E%3C/svg%3E") no-repeat right center;
   }
   ```

1. Simplify the HTML by removing the `<i>` elements

   ```html
   <div class="pagination">
       <div class="pagination-item pagination-item--previous">
           <a href="#/_page1">
               <div class="pagination-item-label">PREVIOUS</div>
               <div class="pagination-item-title">Introduction</div>
           </a>
       </div>
       <div class="pagination-item pagination-item--next">
           <a href="#/_page3">
               <div class="pagination-item-label">NEXT</div>
               <div class="pagination-item-title">Themes</div>
           </a>
       </div>
   </div>
   ```

If you need help generating the SVG chevrons, I've created a Sass function that you will find useful. It's called `_chevron-data-uri.scss` and is available on [GitHub](https://github.com/jhildenbiddle/sass-utilities), [NPM](https://www.npmjs.com/package/sass-utilities), and [unpkg](https://unpkg.com/sass-utilities/) (CDN) as part of a larger `sass-utilities` package.

Thanks!

**Comment by imyelo** [comment]
Wow, this is really helpful. Let me have a try.

**Comment by imyelo** [comment]
I just released version 2.0.1 for this issue. Thank you and hope it works for you. 😄

**Comment by jhildenbiddle** [comment]
Thank you for the quick turnaround, @imyelo! Much appreciated.

The update removes the unnecessary requests with is great. After reviewing the SVG icon implementation, I realized there is a better approach that will better align with the docsify theme in use. Continuing conversation in #3...
